fr-FR/lang/numere.nlng

```diff
@@ -0,0 +1,1186 @@
+# (à votre choix) toute version ultérieure.
+#
+# Ce programme est distribué dans l'espoir qu'il sera utile,
+# but WITHOUT ANY WARRANTY ; without even the implied warranty of
+# MERCHANTABILITY ou FITNESS POUR UNE PURPOSE PARTICULIÈRE.  Voir le
+# GNU General Public License pour plus de détails.
+#
+# Vous devriez avoir reçu une copie de la Licence Publique Générale GNU
+# avec ce programme.  Si ce n'est pas le cas, voir <http://www.gnu.org/licenses/>.
+# ========================================================================
+# LANGUAGE FILE: numere.nlng
+LANGUAGE: FR
+#
+# PARSERFUNCS:
+#======================================
+PARSERFUNCS_LISTFUNC_HEADLINE=Vordefinierte Funktionen
+PARSERFUNCS_LISTFUNC_TYPE_NUM=Numerik
+PARSERFUNCS_LISTFUNC_TYPE_MAT=Matrix und Vektor
+PARSERFUNCS_LISTFUNC_TYPE_TRIGONOMETRIC=Trigonometrie
+PARSERFUNCS_LISTFUNC_TYPE_HYPERBOLIC=Hyperbelfunktionen
+PARSERFUNCS_LISTFUNC_TYPE_POLYNOMIAL=Polynome
+PARSERFUNCS_LISTFUNC_TYPE_STATS=Statistik
+PARSERFUNCS_LISTFUNC_TYPE_STRING=Zeichenketten
+PARSERFUNCS_LISTFUNC_TYPE_LOGIC=Logik
+PARSERFUNCS_LISTFUNC_TYPE_TIME=Zeit
+PARSERFUNCS_LISTFUNC_TYPE_RANDOM=Zufall
+PARSERFUNCS_LISTFUNC_TYPE_PHYSICS=Physik
+PARSERFUNCS_LISTFUNC_TYPE_ANGULAR=Winkel
+PARSERFUNCS_LISTFUNC_TYPE_DISTRIB=Distributionen
+PARSERFUNCS_LISTFUNC_TYPE_LOGARITHMIC=Logarithmen
+PARSERFUNCS_LISTFUNC_TYPE_COORDS=Koordinaten
+PARSERFUNCS_LISTFUNC_TYPE_DRAW=Zeichnungen
+PARSERFUNCS_LISTFUNC_TYPE_DEFINE=Eigene Definitionen
+PARSERFUNCS_LISTFUNC_TYPE_GUI=Graphische Oberflächen
+PARSERFUNCS_LISTFUNC_TYPE_CMPLX=Komplexe Zahlen
+# Funktionentabelle
+DEFINED_FOR_ALL=Definiert für alle reellen oder komplexen Zahlen
+DEFINED_FOR_ALL_C_RANGE_R=Definiert für alle komplexen Zahlen oder für reelle Zahlen im Bereich
+DEFINED_REAL=Definiert für alle reellen Zahlen
+DEFINED_NATURAL=Definiert für natürliche Zahlen
+PARSERFUNCS_LISTFUNC_FUNC_ABS_[NUM]=						abs(x)                                                      VAL      - Montant du nombre réel ou complexe x
+PARSERFUNCS_LISTFUNC_FUNC_ACOSH_[NUM_HYPERBOLIC]=			acosh(x)                                                    VAL      - Areakosinus Hyperbolicus de x. %%DEFINED_FOR_ALL_C_RANGE_R%% [1,inf) (synonyme : "arcosh(x)")
+PARSERFUNCS_LISTFUNC_FUNC_ACOS_[NUM_TRIGONOMETRIC]=			acos(x)                                                     VAL      - Arcuscosinus de x. %%DEFINED_FOR_ALL_C_RANGE_R%% [-1,1] (synonyme : "arccos(x)")
+PARSERFUNCS_LISTFUNC_FUNC_ACSCH_[NUM_HYPERBOLIC]=			acsch(x)                                                    VAL      - Hyperbolicus d'Areakosekan de x. %%DEFINED_FOR_ALL_C_RANGE_R%% [1,inf]
+PARSERFUNCS_LISTFUNC_FUNC_ACSC_[NUM_TRIGONOMETRIC]=			acsc(x)                                                     VAL      - Arkuskosekans de x. %%DEFINED_FOR_ALL_C_RANGE_R%% [-1,1]
+PARSERFUNCS_LISTFUNC_FUNC_AI_[NUM_PHYSICS]=					Ai(x)                                                       VAL      - Fonction Airy Ai(x). %%DÉFINI_RÉEL%%
```

Comment:
`%%DÉFINI_RÉEL%%` should not be translated, because this hinders the correct replacement with `DEFINED_REAL`